java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/CPMapTest.java

```diff
@@ -55,6 +75,16 @@ public void setup() {
             String mapName = "map" + i + "@" + cpGroup;
             mapReferences.add(targetInstance.getCPSubsystem().getMap(mapName));
         }
+
+        operationCounterList = targetInstance.getList(name + "Report");
+    }
+
+    private String[] createValues() {
+        String[] valuesArray = new String[this.valuesPerClient];
```

Comment:
I don't understand the values per client. The values are not per client since 'v' is global. And can we use a better name than just 'v'?

Reply:
I'm not sure how simulator work with this internally but when I played with that then this `v[]` was related to certain simulator thread and global. Originally I used just one array and expected that all threads will use it but it showed that each of them has own unique array of values (that's also the reason why I added `private ISet<String> allValues;` to have access to all possible values).

Reply:
Each thread will get the shared reference to the array; the CPMapTest instance is shared between all threads of a single load generator (so a single client for example).

Reply:
> CPMapTest instance is shared between all threads of a single load generator

For simulator tests we have 50 load generators and each of them is running 3 threads. Hence IIUC if `valuesPerClient=20` we have 1000 unique values. Maybe we can change the name of `valuesPerClient` to `valuesLoadGenerator`.

I can update `v` to `loadGeneratorValues` - `v` was used in original `CpMapTest`.

Reply:
Yes. There will be 1000 unique values. But this could be reduced to 20 unique values if you can make the value creation deterministic. So make sure that every load generator generates exactly the same sequence of values.

I would rename it to 'values'. That is how it is called in the other tests and I don't see a reason why it should be any different here.